SendITapp/api/v1/views/parcelview.py

```diff
@@ -163,7 +143,6 @@ def patch(self, parcelid):
 
         payload = {"Status": "Order Updated"}
```

Comment:
Return the updated parcel.  Also, create a method in your models to do what this patch method is supposed to do. Your doc string shows it's `cancel a request` though I am not sure that's what is happening.